examples/amp-story-subscriptions/amp-story-subscriptions.html

```diff
@@ -249,9 +248,9 @@ <h3>Cultral Experiences</h3>
         </amp-story-grid-layer>
       </amp-story-page>
 
-      <amp-story-subscriptions
-        layout="container"
-      ></amp-story-subscriptions>
+      <amp-story-subscriptions layout='container' price='$0.50/week' title='Information at your fingertips' subtitle-first='Subscribe for unlimited access.'
+        subtitle-second='Cancel anytime.'>
+      </amp-story-subscriptions>
```

Comment:
Please put returns on attributes if there are more than one or two.
It's more readable and a faster way to get an overview of the API.
```suggestion
      <amp-story-subscriptions layout='container' 
        price='$0.50/week' 
        title='Information at your fingertips' 
        subtitle-first='Subscribe for unlimited access.'
        subtitle-second='Cancel anytime.'>
      </amp-story-subscriptions>
```